docs/cli/pushing-data-to-pulse.md

```diff
@@ -8,10 +8,10 @@ To measure the performance of your team, you must send information to Pulse abou
 !!! important
     Consider the following before setting up the integration using the Pulse CLI:
 
-    -   **Make sure that you have an API key** provided by Pulse to identify your organization and authorize you to send data to Pulse.
+    -   [**Make sure that you have an API key**](https://app.pulse.codacy.com/integrations/cli){target=_blank} provided by Pulse to identify your organization and authorize you to send data to Pulse.
```

Comment:
I'm not sure if this syntax for adding the `target` attribute is working, but either way I suggest that we use the [recommended one](https://python-markdown.github.io/extensions/attr_list/):

```suggestion
    -   [**Make sure that you have an API key**](https://app.pulse.codacy.com/integrations/cli){: target="_blank"} provided by Pulse to identify your organization and authorize you to send data to Pulse.
```